src/getting_started.md

```diff
@@ -4,18 +4,31 @@ This guide walks you through the steps needed to get the Alan compiler and runti
 
 ## Installation
 
-Node.js is required, with a minimum version of 10.20.1, to install `alan-compile`, run:
+For users on Windows, MacOS, and Ubuntu Linux with x86-64 processors, you can [download the Alan executable](https://github.com/alantech/alan/releases), extract it into your `$PATH`/`%PATH%` (eg `/usr/local/bin` on MacOS and Ubuntu, and `C:\Windows` on Windows if you don't want to add a new executable path to your environment variable), and you're ready to go!
```

Comment:
Can we explicitly add the ‘mv’ command for Unix that puts ‘alan’ in ‘usr/local/bin’ to not make the user think or in case they are not super familiar with bash/Unix (like a student)?

Reply:
https://github.com/alantech/alan/issues/270

Reply:
Right, but I am just suggesting we add the mv command

Reply:
Ahh, sure, I can add some examples on doing this.